lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -125,7 +129,11 @@ def sanitize(value)
     end
 
     def merge_doc_types(types)
-      Hash[@parsed_hash['signatures'].flatten.map { |param| [ param['name'], param['type'] ] }].merge(types)
+      Hash[tag_params.select { |param| !param['types'].nil? }.map { |param| [ param['name'], param['types'].first ] }].merge(types)
+    end
+
+    def tag_params
+      @parsed_hash['docstring']['tags'].select { |tag| tag['tag_name'] == 'param' }
```

Comment:
I tried more debugging and after VM reboot I'm getting slightly different error at the same place, now the select is not called on String but nil, it seems that it's caused by specific module. The structure is correct but tags is missing

```
pry(#<KafoParsers::PuppetStringsModuleParser>)> @parsed_hash['docstring']
=> {"text"=>"Installs foreman_ansible plugin"}
```

it succeeded on ~8 modules previously. So we probably need to handle nil here.